contracts/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "skipLibCheck": true,
```

Comment:
@nick-bisonai Is this also related to the hardhat issue?

Reply:
I added this line to fix build error which happened after mono repo implementation. it was type some kind of type error

```
../node_modules/@types/mocha/index.d.ts:2642:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'beforeEach' must be of type 'Lifecycle', but here has type 'HookFunction'.

2642 declare var beforeEach: Mocha.HookFunction;
                 ~~~~~~~~~~

  ../node_modules/@types/jest/index.d.ts:34:13
    34 declare var beforeEach: jest.Lifecycle;
                   ~~~~~~~~~~
    'beforeEach' was also declared here.

../node_modules/@types/mocha/index.d.ts:2660:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'afterEach' must be of type 'Lifecycle', but here has type 'HookFunction'.

2660 declare var afterEach: Mocha.HookFunction;
                 ~~~~~~~~~

  ../node_modules/@types/jest/index.d.ts:36:13
    36 declare var afterEach: jest.Lifecycle;
                   ~~~~~~~~~
    'afterEach' was also declared here.

../node_modules/@types/mocha/index.d.ts:2676:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'describe' must be of type 'Describe', but here has type 'SuiteFunction'.

2676 declare var describe: Mocha.SuiteFunction;
                 ~~~~~~~~

  ../node_modules/@types/jest/index.d.ts:37:13
    37 declare var describe: jest.Describe;
                   ~~~~~~~~
    'describe' was also declared here.

../node_modules/@types/mocha/index.d.ts:2697:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'xdescribe' must be of type 'Describe', but here has type 'PendingSuiteFunction'.

2697 declare var xdescribe: Mocha.PendingSuiteFunction;
                 ~~~~~~~~~

  ../node_modules/@types/jest/index.d.ts:39:13
    39 declare var xdescribe: jest.Describe;
                   ~~~~~~~~~
    'xdescribe' was also declared here.

../node_modules/@types/mocha/index.d.ts:2711:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'it' must be of type 'It', but here has type 'TestFunction'.

2711 declare var it: Mocha.TestFunction;
                 ~~

  ../node_modules/@types/jest/index.d.ts:40:13
    40 declare var it: jest.It;
                   ~~
    'it' was also declared here.

../node_modules/@types/mocha/index.d.ts:2725:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'test' must be of type 'It', but here has type 'TestFunction'.

2725 declare var test: Mocha.TestFunction;
                 ~~~~

  ../node_modules/@types/jest/index.d.ts:43:13
    43 declare var test: jest.It;
                   ~~~~
    'test' was also declared here.

../node_modules/@types/mocha/index.d.ts:2732:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'xit' must be of type 'It', but here has type 'PendingTestFunction'.

2732 declare var xit: Mocha.PendingTestFunction;
                 ~~~

  ../node_modules/@types/jest/index.d.ts:42:13
    42 declare var xit: jest.It;
                   ~~~
    'xit' was also declared here.


Found 7 errors in the same file, starting at: ../node_modules/@types/mocha/index.d.ts:2642
```

Reply:
Okay! Thank you!